internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode    int64             = 8     // per host requires 8 cpus for OCS in compact mode
+	CPU               int64             = 6     // per host requires 6 cpus for OCS in minimal and standard mode
+	MemoryCompactMode int64             = 22889 // Memory value provided in MiB for per host (24 GB)
```

Comment:
What do you think of storing this as bytes? perhaps doing `24*conversions.GB` (`GB` is not there yet, but it would be trivial to add it). I found easier to read when they are declared like that rather than having to depend on the comment.